services/wikidataService.ts

```diff
@@ -55,4 +59,23 @@ export const searchTerm = async (term, languageCode) => {
   return search;
 };
 
+export const searchImage = async (term) => {
+  const { search, error } = await wikimediaService.get<any, Response>(
```

Comment:
without interceptor it will be 
```suggestion
  const {data:{ search, error }} = await wikimediaService.get<any, Response>(
```

Reply:
annoying stuff